### ISSUE TITLE
[docs][tutorial] Improve contrast of blog post dates

### ISSUE THREAD

**Issue by GK-Hynes** [gatsbyjs/gatsby#20694]
## Summary
The tutorial uses #bbb as the color for the blog post dates. This only has a contrast ratio of 1.92 and fails to reach WCAG level AA.

## Steps to resolve this issue
Reducing the color's lightness to #767676 would achieve a contrast ratio of 4.54, reaching WCAG AA, while keeping the shade of grey similar.

**Comment by shrey-sachdeva2000** [comment]
Could you please attach a screenshot of the same? @GK-Hynes

**Comment by GK-Hynes** [comment]
Absolutely. I should have done this to begin with. 

Here's the page currently with `color: #bbb` (Contrast ratio 1.92)
![image](https://user-images.githubusercontent.com/28233682/73135176-29fc4900-4037-11ea-8b49-cf913a7f16d8.png)

Here's how it would look if the lightness was reduced to `color: #767676` (Contrast ratio 4.54)
![image](https://user-images.githubusercontent.com/28233682/73135187-44cebd80-4037-11ea-9b4b-12cf83664767.png)

And here's how it would look if the lightness was reduced to `color: #595959`  (Contrast ratio 7)
![image](https://user-images.githubusercontent.com/28233682/73135196-5912ba80-4037-11ea-8554-33af799edab4.png)

I opened PR #20695 to change the color to #767676 but could update that if another color was preferable.

**Comment by ShivamGoyal1899** [comment]
@shrey-sachdeva2000 Is this issue still unresolved? If it is, I can help out. Kindly assign it to me, if it's available.

**Comment by shrey-sachdeva2000** [comment]
Yes, please go for it! @ShivamGoyal1899
